administrator/components/com_fields/helpers/fields.php

```
@@ -472,6 +472,14 @@ public static function prepareForm($context, JForm $form, $data)
 			{
 				$value = $model->getFieldValue($field->id, $field->context, $data->id);
 
+				// Creating the XML form data
+				$formField = $form->getField($field->alias, 'params');
+
+				if ($formField && !is_array($value) && $value !== '' && $value !== null && $formField->forceMultiple)
```

`!empty($data->id)` will throw a notice when the id is not set